src/containers/Profile/style.module.css

```
@@ -0,0 +1,74 @@
+.actionBtn {
+  margin-right: 5px;
+}
+.bgWrapperPerfil{
+  height: 170px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+.cardPerfil{
+  width: 220px;
+  height: 200px;
+}
+.divCard {
+  width: 96%;
+  height: 95%;
+  background-color: white;
+  border-radius: 2px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: static;
+}
```

corrigir ordem alfabética das propriedades (fazer isto em todas as classes)

```suggestion
.divCard {
  align-items: center;
  height: 95%;
  background-color: white;
  border-radius: 2px;
  display: flex;
  flex-direction: column;
  position: static;
  width: 96%;
}
```